app/controllers/emails_controller.rb

```diff
@@ -54,15 +54,22 @@ def save_template
       flash.now[:alert] = t('.saving_failed')
     end
     @event = Event.find(params[:event_id])
-    @templates = EmailTemplate.with_status(get_status)
+    @templates = EmailTemplate.with_status(get_email_template_status)
 
     render :email
   end
 
-  def get_status
+  def get_email_template_status
```

Comment:
Do you mean the documentation stuff? I think we agreed to write documentation for model functions, and this is just a (private) helper function inside the controller